components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/FirefoxAccount.kt

```diff
@@ -105,16 +106,24 @@ class FirefoxAccount internal constructor(
     }
 
     override fun beginOAuthFlowAsync(scopes: Set<String>) = scope.async {
+        // Eventually we should specify this as a param here, but for now, let's
+        // use a generic value (it's used only for server-side telemetry, so the
+        // actual value doesn't matter much)
+        val entryPoint = "android-components"
```

Comment:
nit: could we add this as an optional param in `beginOAuthFlowSync`, so we can update consumers to pass in their own values without waiting on another release of a-c?

Reply:
good idea!